### PR TITLE
SetTrajectory and Field2d options docs

### LINKED ISSUE #1335: Document Glass/Simulation Line Style Options
image

### PR DESCRIPTION
**Pull request by Daltz333**
Close #1335

### CODE REVIEW

File: source/docs/software/advanced-controls/trajectories/sending-trajectories-field2d.rst [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+         :lineno-start: 18
+
+Viewing with Glass
+------------------
+
+.. todo:: link to glass document on modifying field2d lines
+
+The sent trajectory can be viewed with :ref:`Glass <docs/software/dashboards/glass/index:Glass>` through the dropdown :guilabel:`NetworkTables` -> :guilabel:`SmartDashboard` -> :guilabel:`Field2d`.
+
+.. image:: images/sent-trajectory.png
```
Comment (prateekma) [inline source/docs/software/advanced-controls/trajectories/sending-trajectories-field2d.rst:31]: You should select the path option in Field2d here and not a series of robot poses .

**Review by prateekma — CHANGES_REQUESTED** [review]

**Comment by Starlight220** [comment]
Is there a reason that this isn't appended to the existing `Field2d` article?

**Comment by Daltz333** [comment]
It can be. I initially felt it was more relevant to the trajectory following section.

File: source/docs/software/advanced-controls/trajectories/sending-trajectories-field2d.rst (hunk 1 above)
Comment (Daltz333) [reply source/docs/software/advanced-controls/trajectories/sending-trajectories-field2d.rst:31]: That's not actually the default behavior. Should probably document setting it to a path.

**Review by prateekma — APPROVED** [review]

**Review by jasondaming — APPROVED** [review]

**Pull request merged**